.circleci/docker/common/install_conda.sh

```diff
@@ -75,7 +75,7 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   }
 
   # Install PyTorch conda deps, as per https://github.com/pytorch/pytorch README
-  CONDA_COMMON_DEPS="astunparse pyyaml mkl=2021.4.0 mkl-include=2021.4.0 setuptools six"
+  CONDA_COMMON_DEPS="astunparse pyyaml mkl=2021.4.0 mkl-include=2021.4.0 setuptools"
```

Comment:
Still one left in functorch/examples/maml_omniglot/support/omniglot_loaders.py and what about torch._six?

Reply:
`torch._six` can stay, though we probably can eradicate it at some point

Reply:
Also,`examples` might have dependencies that are not covered by CI/CD nor by package dependencies